USAGE.md

```diff
@@ -236,6 +236,8 @@ name: Daniel S. Himmelstein  # mandatory
 initials: DSH  # optional
 orcid: 0000-0002-3012-7446  # mandatory
 twitter: dhimmel  # optional
+mastodon: dhimmel  # optional: mastodon username
+mastodon: mastodon.social  # optional: mastodon server (instance)
```

Comment:
Typo here: `mastodonserver`

Reply:
I think we probably want to use `mastodon-server` as the key versus `mastodonserver` since that's the style [used elsewhere](https://github.com/manubot/manubot/blob/82f25cb8d698a12d69e533686867a8a4afd35c4c/manubot/pandoc/tests/test_cite_filter/input.md#L2-L22) by manubot and pandoc.

Reply:
Noting an experiment I did at https://genomic.social/@dhimmel/109710993689898079

Reply:
@laurentperrinet would you be able to update the `mastodonserver` key to `mastodon-server` and fix line 240? I think that is the last outstanding required change on this PR.